.pre-commit-hooks.yaml

```diff
@@ -6,6 +6,7 @@
   language: script
   files: \.pkr\.hcl$
   pass_filenames: true
+  require_serial: true
```

Comment:
Why do we need serial processing here?

Reply:
Strictly speaking I don't think we need it right now. I mirrored the approach of the pre-commit hook we use that runs `terraform validate` on the basis that I may need to start running `packer init` which I would not want running multiple times at once.

Reply:
If it's not needed, then I say we remove it.  If you find a case where it is needed, then I'm all for keeping it and including an explanation somewhere in this PR.

Reply:
After testing it does not appear to be necessary. I removed it in [d09e717](https://github.com/cisagov/pre-commit-packer/pull/38/commits/d09e7179b21bdf82333bb03bfbc26978b79fd2af) along with another unnecessary option.

Reply:
Thanks for checking on that!